tchannel/schemes/thrift.py

```diff
@@ -135,6 +136,10 @@ def __call__(
             headers=headers, hostport=hostport, encoding='thrift'
         )
 
+        yield self.tchannel.event_emitter.fire(
```

Comment:
any value in using [`tornado.gen.maybe_future`](https://github.com/tornadoweb/tornado/blob/master/tornado/gen.py#L845)?